appengine-deploy/src/main.ts

```diff
@@ -21,7 +21,9 @@ import * as setupGcloud from '../../setupGcloudSDK/dist/index';
 async function run(): Promise<void> {
   try {
     // Add notice
-    core.warning('Thank you for using appengine-deploy Action. GoogleCloudPlatform/github-actions/appengine-deploy has been deprecated, please switch to google-github-actions/appengine-deploy.');
+    core.warning(
+      'Thank you for using appengine-deploy Action. GoogleCloudPlatform/github-actions/appengine-deploy has been deprecated, please switch to google-github-actions/appengine-deploy.',
```

Comment:
For consistency, I want to update this to `deploy-appengine`

Reply:
Done, should `cloudfunctions` in `deploy-cloud-functions` be one word or should I keep it as is?

Reply:
as discussed offline keeping as `deploy-cloud-functions` for readability